### PR TITLE
Attempt Windows, py3.8 arm64 and pytorch 2 builds

### PR DESCRIPTION
**Pull request by stepanhruda**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

I'd like to see the build output/errors for why these are disabled and potentially enable them.

### CODE REVIEW

**Comment by stepanhruda** [comment]
Windows builds don't work because there is no win-64 pytorch in conda-forge, so removing them.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -23,7 +23,7 @@ build:
     - fairseq-validate = fairseq_cli.validate:cli_main
   script: {{ PYTHON }} -m pip install . -vv
   skip: true  # [win]
-  skip: true  # [py==38 and target_platform=='osx-arm64']
+  skip: true  # [py==311]
```
Comment (stepanhruda) [inline recipe/meta.yaml:26]: looks like there is a legit bug in fairseq that 3.11 starts to crash on

**Comment by stepanhruda** [comment]
This passed, can I get a review/merge here? @hadim

**Comment by hmaarrfk** [comment]
sorry i missed this, but what exactly did this do at the time? 

I feel like while we had pinnings in `conda_build_config` they were actually taking place.

In either case, I've merged https://github.com/conda-forge/fairseq-feedstock/pull/34 which should have a similar effect.

Do let me know if there is anything I've missed.

Sorry for not taking a look at this sooner.

Thank you for your help!

**Comment by stepanhruda** [comment]
This removes skipping arm64 builds, which is still not fixed

**Comment by hmaarrfk** [comment]
osx arm builds seem to exist in anaconda.org

image

to be enabled in the latest build.
image

**Comment by stepanhruda** [comment]
python 3.8 + osx-arm64 specifically

**Comment by hmaarrfk** [comment]
understood. you can rebase this, or i opened #36

**Comment by hmaarrfk** [comment]
please update the title as well to reflect that windows support is not added.

**Comment by hmaarrfk** [comment]
I think #36 should be fine. but let me know what you prefer

**Pull request closed without merging**